package.json

```diff
@@ -109,16 +104,16 @@
     "jest": "^24.1.0",
     "lerna": "^3.10.7",
     "lint-staged": "^9.0.0",
-    "lodash": "^4.17.11",
+    "lodash": "^4.17.15",
     "markdown-spellcheck": "^1.3.1",
     "metro-react-native-babel-preset": "^0.55.0",
     "prompt": "^1.0.0",
     "prop-types": "^15.5.10",
     "react": "16.8.3",
     "react-dom": "16.8.3",
-    "react-native": "0.59.0",
+    "react-native": "^0.60.4",
```

Comment:
@tiagoengel Could you please pin this to a specific version: `0.60.4`?

Reply:
done